src/assets/wise5/components/aiChat/ai-chat-bot-message/ai-chat-bot-message.component.html

```diff
@@ -1,10 +1,21 @@
 <div fxLayoutAlign="start start" fxLayoutGap="4px">
-  <mat-icon class="mat-36" color="accent" aria-label="Automated guidance response" i18n-aria-label
-    >smart_toy</mat-icon
-  >
+  <mat-icon
+    *ngIf="computerAvatarImageSrc == null"
+    class="mat-36"
+    color="accent"
+    aria-label="Automated guidance response"
+    i18n-aria-label
+  ></mat-icon>
+  <img
+    *ngIf="computerAvatarImageSrc != null"
+    [src]="computerAvatarImageSrc"
+    class="computer-avatar"
+    aria-label="Automated guidance response"
+    i18n-aria-label
+  />
   <div fxFlex fxLayoutAlign="start">
     <div class="response-text notice-bg-bg">
-      <div class="mat-small secondary-text poster" i18n>AI Chat Bot</div>
+      <div class="mat-small secondary-text poster" i18n>{{ displayNames }}</div>
```

Comment:
Get name directly from computerAvatar?
```suggestion
      <div class="mat-small secondary-text poster" i18n>{{ computerAvatar.name }}</div>
```